libpod/boltdb_state_internal.go

```diff
@@ -566,9 +566,8 @@ func (s *BoltState) addContainer(ctr *Container, pod *Pod) error {
 
 		// Add container to volume dependencies bucket if container is using a named volume
 		for _, vol := range ctr.config.Spec.Mounts {
-			if strings.Contains(vol.Source, ctr.runtime.config.VolumePath) {
+			if len(ctr.runtime.config.VolumePath) > 0 && strings.Contains(vol.Source, ctr.runtime.config.VolumePath) {
 				volName := strings.Split(vol.Source[len(ctr.runtime.config.VolumePath)+1:], "/")[0]
```

Comment:
I think it would be better to say 
```
          if ctr.runtime.config.VolumePath != "" &&
```

Reply:
done.

Reply:
Would you move that check before the loop? Currently, we iterate over all mounts even when VolumePath is empty.

Reply:
@mheon, is the usage of `strings.Containes(...)` intentional? To me it looks like it should be a `strings.HasPrefix(...)` with a break at the end of the branch but as the code continues iterating, I am not sure.

Reply:
@vrothberg I'm going to try and patch this out today... I think I have a better way which doesn't involve slicing up volume paths to check where the container lives.